app/models/spree/credit_card.rb

```diff
@@ -83,12 +83,12 @@ def actions
     end
 
     def can_resend_authorization_email?(payment)
-      payment.pending? && payment.authorization_action_required?
+      payment.pending? && payment.requires_authorization?
```

Comment:
This needs updating, right? It's one state or the other, but not both?

Reply:
Yes, that's right. Good catch 👍